addons/test_website/tests/test_website_page_properties.py

```diff
@@ -14,3 +14,19 @@ def test_website_page_properties_can_publish(self):
 
     def test_website_page_properties_website_page(self):
         self.start_tour('/', 'website_page_properties_website_page', login='admin')
+
+    def test_website_page_properties_ir_ui_view_published(self):
```

Comment:
Of course, this new test cannot pass since we are temporarily disabling the feature. I'll move this whole commit out for now.